pymc/distributions/continuous.py

```diff
@@ -3994,9 +3994,9 @@ def make_node(self, x, h, z):
         x = at.as_tensor_variable(floatX(x))
         h = at.as_tensor_variable(floatX(h))
         z = at.as_tensor_variable(floatX(z))
-        bshape = broadcast_shape(x, h, z)
-        shape = [False] * len(bshape)
-        return Apply(self, [x, h, z], [at.TensorType(aesara.config.floatX, shape=shape)()])
+        shape = broadcast_shape(x, h, z)
+        broadcastable = [] if not shape else [False] * len(shape)
```

Comment:
broadcastable of `False` corresponds to shape of `None`

Reply:
I understand that the intent is more clearly conveyed with `None`, but isn't this conversion handled [here](https://github.com/aesara-devs/aesara/blob/main/aesara/tensor/type.py#L80-L82)?
I am happy to redo this commit anyway with integers/Nones instead of bools whenever possible, but I just wanted to check with you if I am understanding the logic correctly.

Reply:
That's a workaround because types used to be described with True/False flags before shape came around. I think we will eventually deprecate the True/ False, so it's better to do it already.

The other issue was with using var.shape instead of var.type.shape which are not the same usually (one can be a symbolic expression  and the other is always a tuple). Previously, var.broadcastable and var.type.broadcastable were always the same OTOH so it's an understandable mistake.